### PR TITLE
Stable interface for both Animation player and Animated Sprite

### LINKED ISSUE #350: Check / Implement AnimationPlayer and AnimatedSprite support for ESCPlayer
As a game developer using Escoria I'd like to be able to use an easy set of animations for a character using AnimatedSprite frames, but also complex animations using AnimationPlayer - depending on the character.

The current status wether AnimationPlayer and/or AnimatedSprite are fully supported is unclear.

In this issue, the existing feature set should be analyzed. If there's something missing from either feature, implement it.

### PR DESCRIPTION
**Pull request by dploeger**
Fixes #350

### CODE REVIEW

File: addons/escoria-core/game/core-scripts/esc_item.gd [hunk 1]
```diff
@@ -140,6 +138,13 @@ var collision: Node
 var inventory_item: ESCInventoryItem = null setget ,_get_inventory_item
 
 
+# The movable subnode
+var _movable: ESCMovable = null
+
+# The identified animation sprite
+var _animation_sprite: ESCAnimationPlayer = null
```
Comment (StraToN) [inline addons/escoria-core/game/core-scripts/esc_item.gd:145]: Rename this variable to `_animation_player` to make it clear that it is not an AnimationPlayer nor AnimatedSprite, but a structure that can be any of the two.

**Review by StraToN — CHANGES_REQUESTED** [review]

**Comment by StraToN** [comment]
A very clever solution! Great work!

File: addons/escoria-core/game/core-scripts/esc_item.gd (hunk 1 above)
Comment (dploeger) [reply addons/escoria-core/game/core-scripts/esc_item.gd]: Done

**Review by StraToN — APPROVED** [review]

**Comment by StraToN** [comment]
Thanks!

**Pull request merged**